SPECS/kernel-uvm/kernel-uvm.spec

```diff
@@ -145,13 +148,17 @@ find %{buildroot}/lib/modules -name '*.ko' -exec chmod u+x {} +
 %ifarch x86_64
 /lib/modules/%{name}/vmlinux
 %endif
+/boot/kernel-uvm/bzImage
```

Comment:
Thought: to potentially reconsider about location for this file to drop. the vmlinux file is being dropped under the CLH folder. potentially we want to align here?

Reply:
If the UVM kernel doesn't need to be in the boot partition I think it makes sense to move it. The host boots from /boot, UVM is more like a component of kata-shim.

Reply:
Binary moved out of /boot and into same directory as vmlinux.bin